tests/webview/visual/cnx_homepage_books.py

```diff
@@ -0,0 +1,480 @@
+from selenium import webdriver
+from applitools.eyes import Eyes
+from time import sleep
+from dotenv import load_dotenv
+from applitools.eyes import MatchLevel
+import os
+from tests.utils import cnx_deploy_version
+
+load_dotenv(verbose = True)
+
+applitools_key = os.environ['APPLITOOLS_API_KEY']
+url_addr = os.environ['URL_HOME_PAGE']
+App_Name = 'CNX HOME PAGE BOOK-ICONS'
+
+class Books_Home_Page_Layout:
```

Comment:
This isn't in proper class format. When we name classes in Python there are no underscores. This should be named

`class BooksHomePageLayout:`

Additionally, if you want to group your tests within a Class you should implement the tests as methods like so.

```python
class MyAwesomeTest(object):
    def test_my_awesome_page_layout(self):
        print("everything is awesome")
```
You need to refactor this class to look something like this.

Reply:
To further build on this concept we can pass into our tests fixtures that are available to all the other tests. for example.

```python
class MyAwesomeTest(object):
    def test_my_awesome_page_layout(self, selenium, webview_base_url, request):
        page = selenium.get(webview_base_url)        
        
        eyes = Eyes(request.config.applitools_api_key)

      . . .
```

Reply:
Reference in legacy tests for how to organize as a class:

https://github.com/openstax/cnx-automation/blob/master/tests/legacy/ui/test_create_import_publish_module_and_collection.py#L20-L43